modules/core/src/main/scala/org/scalasteward/core/repoconfig/RepoConfig.scala

```diff
@@ -88,7 +89,11 @@ object RepoConfig {
               buildRoots = x.buildRoots |+| y.buildRoots,
               assignees = x.assignees |+| y.assignees,
               reviewers = x.reviewers |+| y.reviewers,
-              dependencyOverrides = x.dependencyOverrides |+| y.dependencyOverrides
+              dependencyOverrides = x.dependencyOverrides |+| y.dependencyOverrides,
+              signoffCommits =
+                if (x.signoffCommits.isDefined || y.signoffCommits.isDefined)
+                  Some(x.signoffCommits.getOrElse(false) || y.signoffCommits.getOrElse(false))
+                else None
```

Comment:
```suggestion
              signoffCommits = x.signoffCommits.orElse(y.signoffCommits)
```
I thinks this should be identical and simpler